options-manager.js

```diff
@@ -195,6 +196,10 @@ function preprocess(opts) {
 	opts = mergeWithPkgConf(opts);
 	opts = normalizeOpts(opts);
 	opts.ignores = DEFAULT_IGNORE.concat(opts.ignores || []);
+
+	const gitignore = parseGitignore('.gitignore');
+	opts.ignores = opts.ignores.concat(gitignore || []);
```

Comment:
Create a function instead, `getIgnores` or something, that we then can export like we've done with the other ones.